GliaWidgets/Sources/View/Common/Alert/AlertStyle.RemoteConfig.swift

```diff
@@ -0,0 +1,117 @@
+import UIKit
+
+// MARK: - Internal
```

Comment:
I would like to align this `MARK` with other places, so perhaps we need to review team agreement about usage and formatting code.
would it make sense @gersonnoboa  @igorkravchenko  @EgorovEI ?

Reply:
I also like MARK's usage

Reply:
I agree using it makes sense, I think @yurii-glia  is talking about having team agreement on how/when to use it to have same structure in the code base. 
If so, then it makes sense to create ticket for such agreement.

Reply:
I propose not using privately declared methods at all, anywhere, if possible. Instead, use private extensions. We can then utilize MARK by clearly differentiating where public, internal, and private methods are declared and found. This keeps the code clean and organized.